### DISCUSSION TITLE
Wiki.js in Microsoft Teams desktop app's Tab

### DISCUSSION THREAD

**Discussion by Mukhammadjon-Jalolov** [requarks/wiki#7169] [Error / Bug Report]
Hello everyone!
I have a wiki.js running on Azure with Azure AD authentication. And now I have uploaded a manifest.json file to Microsoft Teams Tab and it is showing my wiki.js app well. So basically I need a wiki.js showing on my Teams tab with authentication.
However, any of the Authentication is NOT working (both local and Azure AD). It works fine on a browser. What could be reason for that and how that can be fixed? I appreciate any response/solution to this case. Thank you.
![teams tab](https://github.com/requarks/wiki/assets/38403832/8f1bab9e-27b5-4080-aa1e-022fd4775831)

**Comment by tim-elmer** [comment]
Probably related to https://github.com/requarks/wiki/discussions/5912.

**Reply by Mukhammadjon-Jalolov** [reply]
Thank you
